modules/ca/output.tf

```
@@ -5,3 +5,7 @@ output "bucket" {
 output "host" {
   value = "${aws_service_discovery_service.ca_service_discovery.name}.${var.local_dns_namespace.name}"
 }
+
+output "security_group" {
```

```suggestion
output "security_group_id" {
```
nitpicky, but I've been burned thinking this kinda thing was an arn or name or whatever other identifier aws wants to use.

---

Good point! Updated. Thanks!